lib/reply.js

```diff
@@ -267,25 +259,30 @@ const INVALID_TRAILERS = new Set([
 
 Reply.prototype.trailer = function (key, fn) {
   key = key.toLowerCase()
+
   if (INVALID_TRAILERS.has(key)) {
     throw new FST_ERR_BAD_TRAILER_NAME(key)
   }
+
   if (typeof fn !== 'function') {
     throw new FST_ERR_BAD_TRAILER_VALUE(key, typeof fn)
   }
-  if (this[kReplyTrailers] === null) this[kReplyTrailers] = {}
+
+  this[kReplyTrailers] = this[kReplyTrailers] ?? {}
   this[kReplyTrailers][key] = fn
+
   return this
 }
 
 Reply.prototype.hasTrailer = function (key) {
-  if (this[kReplyTrailers] === null) return false
-  return this[kReplyTrailers][key.toLowerCase()] !== undefined
+  return !!this[kReplyTrailers]?.[key.toLowerCase()]
 }
 
 Reply.prototype.removeTrailer = function (key) {
-  if (this[kReplyTrailers] === null) return this
-  this[kReplyTrailers][key.toLowerCase()] = undefined
+  if (this[kReplyTrailers] !== null) {
```

Comment:
I think I add it as a early exit approach.
The readability didn't change a lot when there is few line of code.

Reply:
:ok_hand: